docker-integrations/tendermint-python/src/main.py

```diff
@@ -0,0 +1,31 @@
+import concurrent.futures
+import sys
+
+import grpc
+
+# Add the parent folder to the module search path
+parent_folder = "src/protos"
+sys.path.append(parent_folder)
+
+from protos import runtime_pb2_grpc
+from server import TendermintServer
+
+IP_ADDR = "runtime:50051"
```

Comment:
This does not make it available to run locally at `0.0.0.0`. I'd say let's get the DNS name as an environment variable but its default should be `0.0.0.0`. This env var can be set in the Dockerfile as `ENV HOSTNAME runtime`.

Reply:
Set it as env variable in docker file and used it in the code